src/components/common/BusinessContactInfo.vue

```diff
@@ -202,10 +202,15 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
 
   @Watch('initialValue', { deep: true, immediate: true })
   private onContactPropValueChanged (): void {
-    this.contact = this.initialValue
+    this.contact = { ...this.initialValue }
+
+    // If email exists, set confirmEmail to match
+    if (this.contact.email) {
+      this.contact.confirmEmail = this.contact.email
+    }
   }
 
-  @Watch('contact', { deep: true, immediate: true })
```

Comment:
Please test this change with a different type of filing (since this is a common component).

Reply:
Please test this before merging.